src/test/java/com/github/princesslana/eriscasper/gateway/TestGateway.java

```diff
@@ -1,12 +1,12 @@
 package com.github.princesslana.eriscasper.gateway;
 
+import static org.mockito.BDDMockito.any;
```

Comment:
Why are we importing this in a static manner? We use it once, I don't think it deserves its own import.

Reply:
For me importing it is consistent with how we use `BDDMockito`. Having `BCCMockito` in the fluent API ruins its flow

`given(mockRxWebSocket.send(any()))`
`given(mockRxWebSocket.send(BDDMockito.any()))`

FWIW, I'm usually against static imports, but BDDMockito's API gets very noisy without them.

Reply:
Gotcha, it's more nitpicking and if it's less noisy then it's worthwhile.